src/custom-element.js

```diff
@@ -1348,6 +1348,25 @@ function createBaseCustomElementClass(win) {
       }
     }
 
+    /**
+     * Get the consent policy to follow.
+     * @return {?string}
+     */
+    getConsentPolicy_() {
```

Comment:
A comment here is that no element will be able to say I don't want to be blocked by amp-consent now. Even `<amp-consent>` it self can be blocked by `data-block-on-consent` : ( Which eventually lead to a deadlock. 
We can either ignore the case, I don't believe anyone will put the `data-block-on-consent` attribute on the `<amp-consent>` component.
Or we can introduce another `#shouldBeBlockedByConsent` method.  Any ideas?

Reply:
can you just use `AmpConsent.getConsentPolicy()` to force it to never block? (do we need a an `_never` policy preset?)

Reply:
I think it's fine.

Reply:
On second thought I think this is fine. 
What we want to provide here is to allow publisher to make the final decision on consent blocking behaviors. I think it is the publisher's responsibility to use the feature correctly, and not introduce deadlock.

@jpettitt  isn't `_never` preset equals to not adding `data-block-on-consent`. I'm open to more presets, but I'll add them upon requests.

Reply:
SGTM